docs/UserGuide.md

```diff
@@ -171,23 +171,33 @@ VirusTracker data saves in the hard disk automatically after any command that ch
 
 ### Adding a visit: 
 
-Adds a visit by the personId, location of visit and date of visit
+Adds a visit by the personId, locationId and date of visit.
 
-Format: `add personId location date`
+Format: `addVisit PersonId LocationId d/DATE`
```

Comment:
I think for this, referencing the index rather than id is required. I believe the current master branch has an example of this.